ArcGISRuntimeSDKQt_CppSamples/Maps/ShowLocationHistory/ShowLocationHistory.cpp

```diff
@@ -0,0 +1,131 @@
+}
+
+ShowLocationHistory::ShowLocationHistory(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_map(new Map(Basemap::darkGrayCanvasVector(this), this)),
+  m_locationHistoryOverlay(new GraphicsOverlay(this)),
+  m_locationHistoryLineOverlay(new GraphicsOverlay(this))
+{
+  // graphics overlay for displaying the trail
+  SimpleLineSymbol* locationLineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle::Solid, Qt::green, 2, this);
+  m_locationHistoryLineOverlay->setRenderer(new SimpleRenderer(locationLineSymbol, this));
+  m_locationHistoryLineGraphic = new Graphic(this);
+  m_locationHistoryLineOverlay->graphics()->append(m_locationHistoryLineGraphic);
+
+  // graphics overlay for showing points
+  SimpleMarkerSymbol* locationPointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle::Circle, Qt::red, 3, this);
+  m_locationHistoryOverlay->setRenderer(new SimpleRenderer(locationPointSymbol, this));
+
+  m_polylineBuilder = new PolylineBuilder(SpatialReference::webMercator(), this);
+}
+
+void ShowLocationHistory::handleLocationChanges()
+{
+  m_simulatedLocationDataSource = new SimulatedLocationDataSource(this);
+
+  PolylineBuilder polylineBuilder(SpatialReference::webMercator());
```

Comment:
you have a member polyline builder (`m_polylineBuilder`) above - do we need both?

Reply:
removed this polyline builder following Lucas's suggestion to construct the polyline from a JSON string instead